dissect/target/plugins/os/unix/linux/android/_os.py

```diff
@@ -66,5 +55,19 @@ def version(self) -> str:
     def os(self) -> str:
         return OperatingSystem.ANDROID.value
 
-    def users(self) -> Iterator[UnixUserRecord]:
-        raise NotImplementedError()
+    @export(record=EmptyRecord)
```

Comment:
This addition would temporarily fix https://github.com/fox-it/dissect.target/issues/421.